gitops/generate_build.go

```diff
@@ -182,22 +187,35 @@ func normalizeOutputImageURL(outputImage string) string {
 	// push to quay.io/foo/bar:git-revision
 
 	if strings.Contains(outputImage, ":") {
+		// do not allow use of the default registry and a different user's tag
+		if strings.HasPrefix(outputImage, DefaultImageRepo) {
```

Comment:
nit: should not we allow to redefine this by `IMAGE_REPOSITORY` env variable? It would make the feature more universal and easier for test.

Reply:
In hindsight that makes total sense @Michkov - I will make the update later this morning.

Perhaps this PR should even make some minor tweaks to https://github.com/redhat-appstudio/application-service/#setting-up-the-appstudio-build-service-environment to capture what we are doing with this PR.  If you have some immediate opinion on that, of course let me know.  Otherwise, I'll think about it some more while I'm making the change and then report back here.

thanks

Reply:
updates pushed @Michkov 

in the end, as https://github.com/redhat-appstudio/application-service/#setting-up-the-appstudio-build-service-environment did not previously get into the nuances `mytag-git-revision` vs. `git-revision` I left the README alone.

PTAL - thanks